samples/tab-approval/csharp/README.md

```diff
@@ -0,0 +1,139 @@
+﻿# Tab request approval with activity feed notification.
+
+This sample shows a feature where:
+1)Requester: Can request for any task to the manager by sending activity feed notification and can see his request status.
+2)Manager : Can see the pending request by user on the click of activity feed notification and can approve or reject the request.
+
+User Persona:
+
+- Send request to the manger for task.
+
+![Request from user](TabRequestApproval/Images/TaskRequest.png)
+
+- Request status
+
+![Request status](TabRequestApproval/Images/RequestStatus.png)
+
+Manager Persona:
+
+- Activity feed notification of request.
```

Comment:
Activity feed notification of approval request.

Reply:
updated